interoperability_report.py

```diff
@@ -0,0 +1,655 @@
+        additional_info_sub = information_subscriber.replace('\n', '<br>')  
+
+        case.result = [Error(f'<strong> Publisher 1 expected code: </strong> {expected_code_pub1}; <strong> Code found: </strong> {code[1]} <br> \
+                               <strong> Publisher 2 expected code: </strong> {expected_code_pub2}: <strong> Code found: </strong> {code[2]} <br> \
+                               <strong> Subscriber expected code: </strong> {expected_code_sub}; <strong> Code found: </strong> {code[0]} <br>\
+                              <strong> Information Publisher 1: </strong>  <br> {additional_info_pub1} <br>\
+                              <strong> Information Publisher 2: </strong>  <br> {additional_info_pub2} <br>\
+                              <strong> Information Subscriber: </strong>  <br> {additional_info_sub}' 
+                          )
+                      ]
+
+    # Delete the temporal files
+    os.remove('log_pub_1.txt')
+    os.remove('log_pub_2.txt')
+    os.remove('log_sub.txt')
+
+class Arguments:
+    def parser():
+        parser = argparse.ArgumentParser(
+            description='Interoperability Test',
+            add_help=True)
+
+        gen_opts = parser.add_argument_group(title='general options')
+        gen_opts.add_argument('-P', '--publisher',
+            default=None,
+            required=True,
+            type=str,
+            choices=['connext611', 'opendds321', 'connext5.2.3'],
+            metavar='publisher_name',
+            help='Publisher Shape Application')
+        gen_opts.add_argument('-S', '--subscriber',
+            default=None,
+            required=True,
+            type=str, 
+            choices=['connext611', 'opendds321', 'connext5.2.3'],
+            metavar='subscriber_name',
+            help='Subscriber Shape Application')
+        
+        optional = parser.add_argument_group(title='optional parameters')
+
+        optional.add_argument('-v','--verbose',
+            default=False,
+            required=False,
+            action='store_true',
+            help='Print more information to stdout.')
+
+        out_opts = parser.add_argument_group(title='output options')
+
+        out_opts.add_argument('-f', '--output-format',
+            default='junit',
+            required=False,
+            type=str,
+            choices=['junit', 'csv', 'xlxs'],
+            help='Output format.')
+        out_opts.add_argument('-e', '--extend',
+            default=False,
+            required=False, 
+            action='store_true',
+            help='Save the results in a report that exists previously.')
+        out_opts.add_argument('-o', '--output-name',
+            required=False, 
+            metavar='filename',
+            type=str,
+            help='Report filename.')
+        
+        return parser
+    
+   
+
+def main():
+
+    parser = Arguments.parser()
+    args = parser.parse_args()
+
+    gen_args = {
```

Comment:
gen_args is a bit weird, this represents the options of the script, right? how about `options`?

Reply:
done, i put gen_args because it was the name they used in the python file we had about arguments parser